src/components/staticcard/StaticCard.jsx

```diff
@@ -0,0 +1,79 @@
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import StaticCard1 from "../../assets/static_card_image1.png";
+import StaticCard2 from "../../assets/static_card_image2.png";
+import StaticCard3 from "../../assets/static_card_image3.png";
+import "./staticcard.css";
+
+const StaticCard = () => {
+  return (
+    <div className="container">
+      <div className="row justify-content-center">
+        <div className="col-8">
+          <div className="row mt-5 mb-5">
+            <div className="col-12 col-lg-4 d-flex justify-content-center static-cards">
+              <div className="card shadow">
+                <img
+                  src={StaticCard2}
+                  alt="cardimage1"
+                  className="card-img-top"
+                />
+                <div className="card-body">
+                  <h2 className="card-title">Rent a house</h2>
+                  <p className="card-text">
+                    We’re creating a seamless online experience – from shopping
+                    on the largest rental network, to applying, to paying rent.
+                  </p>
+                  <div className="static-card-button">
+                    <Button className="btn btn-link">Find rentals</Button>
+                  </div>
+                </div>
+              </div>
+            </div>
```

Comment:
This piece of code seems to be repeated in this component. Can you extract it into a separate component and pass the values that it needs through props?
For example, the first one could have a `title` prop set to "Buy a house",  the second one's title prop will have value "Sell a house", etc. Other props could include: "description", "callToActionButtonText", and "image"